package.json

```diff
@@ -28,6 +28,9 @@
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
     "date-fns": "^2.30.0",
+    "embla-carousel": "^8.1.5",
+    "embla-carousel-autoplay": "^8.1.5",
+    "embla-carousel-react": "^8.1.5",
```

Comment:
PR #207  has an alternative implementation of the carousel without the additional packages. Otherwise, you've done a good job.

Reply:
The PR has been merged so you can view the changes on the `Dev` branch.

Reply:
Okay, I have removed the packages now using the existing implemetation